exercises/trampoline/solution/solution.js

```diff
@@ -1,19 +1,20 @@
 function repeat(operation, num) {
   return function() {
-    if (num <= 0) return
-    operation()
-    return repeat(operation, --num)
-  }
+    if (num <= 0) { return; }
+    operation();
+    return repeat(operation, --num);
+  };
 }
 
 function trampoline(fn) {
   while(fn && typeof fn === 'function') {
-    fn = fn()
+    return fn();
```

Comment:
ahh - yea my bad!! Really wondering what my past self was thinking on this one... I'll take a look at why tests consider themselves to be passing with this change!